### PR TITLE
Fix a few design issues

### PR DESCRIPTION
**Pull request by eddieantonio**
Resolves a few bullet points in #292.

### CODE REVIEW

File: src/css/variables.css [hunk 1]
```diff
@@ -53,7 +54,7 @@
   --branding-font-size:             1.50em; /* the title, e.g., "itwêwina" */
   --branding-subtitle-font-size:    0.75em; /* the subtitle, e.g., "Plains Cree Dictionary" */
   --definition-title-size:          2.00em;
-  --tooltip-icon-size:              2rem; /* size of the question mark or the exclamation mark. Kept the same across the app */
+  --tooltip-icon-size:              0.75em; /* size of the ℹ️  button or 🔊 button. Kept the same across the app */
```
Comment (Madoshakalaka) [inline src/css/variables.css]:
```suggestion
  --tooltip-icon-size:              0.75em; /* size of the ℹ️  button or 🔊 button */
```

I used `em` at first too, then it turned out the icon beside the entry title and the preverb breakdown has different sizes. It looked kinda bad cuz they are not uniform. So I changed it to `rem` and added _Kept the same across the app_

Comment (Madoshakalaka) [reply src/css/variables.css]: but it's ok if em produces the same icon size, or it's simply different aesthetics

Comment (Madoshakalaka) [reply src/css/variables.css]: Screenshot from 2020-02-21 14-57-05

so yeah, one is bigger than the other. I guess it's OK? The mockup on Zeplin asked for the same size though.

Comment (eddieantonio) [reply src/css/variables.css]: You're right:

Screen Shot 2020-02-21 at 2 58 25 PM

That _does_ look kinda dumb. `rem`s it is!

File: src/css/variables.css [hunk 2]
```diff
@@ -54,7 +54,7 @@
   --branding-font-size:             1.50em; /* the title, e.g., "itwêwina" */
   --branding-subtitle-font-size:    0.75em; /* the subtitle, e.g., "Plains Cree Dictionary" */
   --definition-title-size:          2.00em;
-  --tooltip-icon-size:              0.75em; /* size of the ℹ️  button or 🔊 button. Kept the same across the app */
+  --tooltip-icon-size:              0.75em; /* size of the ℹ️  button or 🔊 button */
```
Comment (Madoshakalaka) [inline src/css/variables.css]:
```suggestion
  --tooltip-icon-size:              0.75rem; /* size of the ℹ️  button or 🔊 button. Kept the same across the app*/
```

dErP

Comment (Madoshakalaka) [reply src/css/variables.css]: wait, 0.75rem is actually too small. Please bumpt it up a bit

**Pull request merged**